src/legacy/core_plugins/vis_type_vislib/public/vislib/helpers/point_series/_get_aspects.test.ts

```diff
@@ -17,45 +17,45 @@
  * under the License.
  */
 
-import expect from '@kbn/expect';
-import { getAspects } from '../_get_aspects';
+import { getAspects } from './_get_aspects';
+import { Dimension, Dimensions, Aspect } from './point_series';
+import { Table, Row } from '../../types';
 
 describe('getAspects', function() {
-  let table;
-  let dimensions;
+  let table: Table;
+  let dimensions: Dimensions;
 
-  function validate(aspect, i) {
-    expect(aspect)
-      .to.be.an('object')
-      .and.have.property('accessor', i);
+  function validate(aspect: Aspect, i: string) {
+    expect(aspect).toEqual(expect.any(Object));
+    expect(aspect).toHaveProperty('accessor', i);
   }
 
-  function init(group, x, y) {
+  function init(group: number, x: number | null, y: number) {
     table = {
       columns: [
-        { id: '0', title: 'date' }, // date
-        { id: '1', title: 'date utc_time' }, // date
-        { id: '2', title: 'ext' }, // extension
-        { id: '3', title: 'geo.src' }, // extension
-        { id: '4', title: 'count' }, // count
-        { id: '5', title: 'avg bytes' }, // avg
+        { id: '0', name: 'date' }, // date
+        { id: '1', name: 'date utc_time' }, // date
+        { id: '2', name: 'ext' }, // extension
+        { id: '3', name: 'geo.src' }, // extension
+        { id: '4', name: 'count' }, // count
+        { id: '5', name: 'avg bytes' }, // avg
       ],
-      rows: [],
-    };
+      rows: [] as Row[],
+    } as Table;
 
     dimensions = {
-      x: { accessor: x },
-      y: { accessor: y },
-      series: { accessor: group },
-    };
+      x: { accessor: x } as Dimension,
+      y: [{ accessor: y } as Dimension],
```

Comment:
According to [this](https://github.com/elastic/kibana/blob/aab3dffefd1807a191d9d466a79eb079677f2277/src/plugins/visualizations/public/legacy/build_pipeline.ts#L138) and [this](https://github.com/elastic/kibana/blob/aab3dffefd1807a191d9d466a79eb079677f2277/src/plugins/visualizations/public/legacy/build_pipeline.ts#L456) `y` is always be an array.